src/spartan_app.erl

```diff
@@ -110,6 +111,35 @@ start_tcp_listener(IP) ->
         spartan_tcp_handler,
         []).
 
+-spec(maybe_start_http_listener() -> ok).
+maybe_start_http_listener() ->
+    case spartan_config:http_enabled() of
+        true ->
+            IPs = spartan_config:bind_ips(),
```

Comment:
I don't see any problems. HTTP API and DNS TCP API use different TCP ports. What could be wrong with `spartan_config:bind_ips()`?